src/sentry/templates/sentry/integrations/msteams-expired-link.html

```diff
@@ -0,0 +1,14 @@
+{% extends "sentry/bases/modal.html" %}
+
+{% load i18n %}
+
+{% block title %}{% trans "Microsoft Teams Expired Link" %} | {{ block.super }}{% endblock %}
+{% block wrapperclass %}narrow auth{% endblock %}
+
+{% block main %}
+  <div class="align-center">
+    <p>
+      {% trans "Your link has expired. Please retry again." %}
```

Comment:
@NisanthanNanthakumar could we put something a bit more descriptive? Like:
```
Your link is expired. Please try re-linking your identity again by interacting with a Sentry notification in Microsoft Teams.
```

Reply:
@scefali yeah that's better wording